test/helpers.ts

```diff
@@ -11,7 +11,9 @@ import {
 	getScopedSelector,
 	getLatestVersionTag,
 	preventPrCommitLinkLoss,
-	prCommitUrlRegex
+	prCommitUrlRegex,
+	prCompareUrlRegex,
+	preventPrCompareLinkLoss
```

Comment:
```suggestion
```
This was imported 2 times

Reply:
It is done only once.

Reply:
I think you mistaken preventPrCommitLinkLoss ?

Reply:
Your right!